test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,29 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.